nemo_skills/mcp/servers/python_tool.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from dataclasses import dataclass
+from pydantic import Field
+from typing import Annotated, Optional
+from mcp.server.fastmcp import FastMCP
+from nemo_skills.code_execution.sandbox import LocalSandbox
+
+logger = logging.getLogger(__name__)
+
+mcp = FastMCP(name="python_tool")
+
+sandbox = LocalSandbox()
```

Comment:
We need a way from the launch arguments for the sandbox server arguments.

Reply:
Specifically, I am talking about this kind of a structure:

https://github.com/NVIDIA/NeMo-Skills/blob/8214c42a14f8d815eb100c920d4bb33f6e9a5d01/nemo_skills/inference/generate.py#L265

Reply:
Understood and working on it. The crux and decision points lie in actually passing the config to the tool.